app/views/components/widget/_fba2.js.erb

```diff
@@ -416,22 +386,6 @@ function FBAform(d, N) {
 				d.body.prepend(this.skipLink);
 			}
 		},
-		// Used when in a modal
-		loadDialog: function() {
-			d.dispatchEvent(new Event('onTouchpointsModalOpen'));
```

Comment:
I haven't figured out how to retain the `onTouchpointsModalOpen` and `onTouchpointsModalClose` events. How important are these events?

Reply:
the events should remain, if possible

https://github.com/GSA/touchpoints/wiki/Analytics-Events

Reply:
I restored `onTouchpointsModalOpen`, the easy one.

Reply:
I will create a ticket to try to restore `onTouchpointsModalClose`.